packages/backend/src/models/GroupsModel.ts

```diff
@@ -194,31 +197,81 @@ export class GroupsModel {
         await this.database('groups').where('group_uuid', groupUuid).del();
     }
 
-    async addGroupMember(
+    async maybeAddGroupMember(
```

Comment:
Rename so it's clear it might fail but won't throw an error.

Reply:
should we add a deprecated tag ? or maybe replace its usage with the new method and remove this one.